test/app_test.py

```diff
@@ -415,7 +416,8 @@ def test_app_create_bad_environment_name_error(client):
         ):  # TODO: why isn't environment_name an argument to app.run?
             pass
 
-    assert len(asyncio.all_tasks(synchronizer._loop)) == 1  # no trailing tasks, except the `loop_inner` ever-task
+    if sys.platform != "win32":  # proactor event loop has long lived helper tasks
```

Comment:
Is this still needed if we set `WindowsSelectorEventLoopPolicy` as the event loop policy?

Reply:
Nope, but I'd like to keep it since I intend to revert the event loop policy change for windows and this is required for that (since the test still runs on other platforms and windows is documented to leak networking tasks I'd say it's fine)